docs/course_materials/exercises/07_Exceptions/homework.md

```diff
@@ -0,0 +1,24 @@
+# Exceptions - HomeWork 
+
+### ex1_exceptions.py
+The program asks for numeric number from the user input. 
+If the user doesn’t enter a numeric input the program normally would crash. 
+Create this program and using by try and except (the error is ValueError).
+
+
+### ex2_exceptions.py
+Adding exceptions error to this code (adding try and except).
```

Comment:
`
Create a new file ex2_exceptions.py and copy the following code to the file: 

<pre><code data-trim data-noescape class="hljs">
  a = 3
    if a < 4
        b = a/(a-3)
    print "Value of b = ", b
</code></pre>

Add the following error exceptions to the code: 
ZeroDivisionError  
NameError 




`